### PR TITLE
Add custom title and icon

### PR DESCRIPTION
**Pull request by ErdbeerbaerLP**
image

### CODE REVIEW

**Comment by Selim042** [comment]
Why crash the game intentionally?

**Comment by ErdbeerbaerLP** [comment]
Just for fun, it has a 0.1% chance, I maybe add Config file for it

**Comment by antoinegag** [comment]
I think we should avoid anything that crash the game on purpose. It's just obnoxious for players and not really fun.

Fake crashes would be funny tho!

**Comment by ErdbeerbaerLP** [comment]
How to make those fake crashes? I later want to make bluescreen item making fake bsod.

**Comment by LegendaryGeek** [comment]
If you want to fake a bsod, make shure to force mc into windowed so people dont worry as much

File: src/main/java/com/mcmoddev/communitymod/erdbeerbaerlp/RandomCrashes.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	int attempts = 0;
+	final Random r = new Random();
+	@SubscribeEvent
+	public void onIngameTick(TickEvent.PlayerTickEvent ev) {
+		if(i >= TimeUnit.MINUTES.toSeconds(1)*20) { //Make sure to execute it only every minute
+			i=0;
+			attempts++;
+			int number = r.nextInt(1000); 
+			if(number == 1) { 
+				someCrashingMethod();
```
Comment (antoinegag) [inline src/main/java/com/mcmoddev/communitymod/erdbeerbaerlp/RandomCrashes.java]: Sub mods should not crash the game on purpose

**Review by antoinegag — CHANGES_REQUESTED** [review]

**Comment by ErdbeerbaerLP** [comment]
I remove it soon (~30-50 minutes)

**Comment by ErdbeerbaerLP** [comment]
Removed crashing code

**Comment by KiriCattus** [comment]
This formatting of code makes me hurt bad... Please clean it up a little xD

**Comment by ErdbeerbaerLP** [comment]
Will do, currently working on something and that will clean up a little too

**Comment by JamiesWhiteShirt** [comment]
Preface: Core mod contributions are allowed in my book, but they must abide by the rules and be reviewable.

All this ASM and its surrounding formatting is an unreadable mess which I will not review. It is practically obfuscated.

I am rejecting this for multiple reasons, but the primary one is this principle: A mod should not try to disguise the game as something it is not. We are modding the game, and the game's title stays. If "Minecraft" was still in the title I could have accepted it, but there are still a number of other issues.

**Comment by ErdbeerbaerLP** [comment]
I still have the ability to change the title to say Minecraft instead of MMD Community Mod

**Comment by JamiesWhiteShirt** [comment]
If you can do it in a readable fashion, open a new PR with whatever is left.

**Comment by ErdbeerbaerLP** [comment]
Is commenting what it does enough or do I need to redirect code execution?

**Pull request closed without merging**